### DISCUSSION TITLE
Improvements after tracking of a Issue with a long title

### DISCUSSION THREAD

**Discussion by edelciomolina** [community/community#4745] [Projects and Issues] [answered]
After tracking a issue that has a long title the layout is showing not so good. 

The issue can be reproduced after create a new Issue with the title below and create a tracking into body of another Issue.
> Bacon ipsum dolor amet burgdoggen tenderloin beef ribs chislic. Jerky meatball swine bacon drumstick, frankfurter porchetta kielbasa turducken ground round. Kielbasa pork chop drumstick biltong

Here an example before the tracking

![image](https://user-images.githubusercontent.com/4604923/126525409-e65b4a82-1e28-4050-a7d4-be56916e3390.png)

And after tracking link

![image](https://user-images.githubusercontent.com/4604923/126525419-af3cdf01-96d9-41af-a4a2-a24e254441fb.png)

I suppose all that's missing is the definition of a `display: initial` or `display: inline` in the `.comment-body .issue-link` style.

![image](https://user-images.githubusercontent.com/4604923/126525427-b49a21fb-8b36-4811-b610-9120f65cc701.png)

**Comment by leandrosimoes** [comment, marked as answer]
I was having the same issue, but it was happening because of [this](https://chrome.google.com/webstore/detail/github-issue-link-status/nbiddhncecgemgccalnoanpnenalmkic) chrome extension. Check if you are not using some extension that is causing this behavior.

**Comment by edelciomolina** [comment]
@leandrosimoes you hit the nail on the head!
I was using that same extension. The issues are now colorized by default, so it is now unnecessary.
Thanks! 😉
